src/ert/dark_storage/common.py

```diff
@@ -1,31 +1,33 @@
 from typing import Any, Dict, List, Optional, Sequence, Union
+from uuid import UUID
 
 import pandas as pd
 
 from ert.config import EnkfObservationImplementationType
 from ert.libres_facade import LibresFacade
-from ert.storage import EnsembleReader
+from ert.storage import EnsembleReader, StorageReader
 
 
-def ensemble_parameter_names(res: LibresFacade) -> List[str]:
-    return res.gen_kw_keys()
+def ensemble_parameter_names(storage: StorageReader, ensemble_id: UUID) -> List[str]:
+    return storage.get_ensemble(ensemble_id).get_gen_kw_keyset()
```

Comment:
We should probably use the `experiment` directly here, as we register all the parameters there:
```python
ensemble = storage.get_ensemble(ensemble_id)
return [val for val in ensemble.experiment.parameter_configuration.values() if isinstance(val, GenKwConfig)]
```

Ideally we would like to avoid the `isinstance` check, but as we only support GenKw for plotting it must stay for now.